<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLineEdit" name="firstNameEdit">
       <property name="placeholderText">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lastNameEdit">
       <property name="placeholderText">
        <string>Apellido</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="usernameEdit">
       <property name="placeholderText">
        <string>Usuario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="adminSpinBox">
       <property name="maximum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="passwordEdit">
       <property name="styleSheet">
        <string notr="true">text-color:black;</string>
       </property>
       <property name="placeholderText">
        <string>Contraseña</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="confirmPasswordEdit">
       <property name="placeholderText">
        <string>Confirmar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    color: #333;
}
QCheckBox::indicator {
    width: 18px;
    height: 18px;
}</string>
       </property>
       <property name="text">
        <string>Mostrar Contraseña</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="entityEdit">
       <property name="styleSheet">
        <string notr="true">text-color:black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Entidad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addressEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Dirección de la Entidad</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="rejectButton">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>UserDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rejectButton</sender>
   <signal>clicked()</signal>
   <receiver>UserDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
